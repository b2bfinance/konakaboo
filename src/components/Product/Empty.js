import React from 'react';
import styled from 'styled-components';
import Col from '../Col';
import Row from '../Row';

export const Wrapper = styled(Col)`
  background-color: ${props => props.theme.productEmptyBackground};
  margin: 2rem auto;
  padding: 2rem;
`;

export default () => (
  <Row>
    <Wrapper phone="100" desktop="35">
      <p>There are no product matching your requirements.</p>
    </Wrapper>
  </Row>
);
