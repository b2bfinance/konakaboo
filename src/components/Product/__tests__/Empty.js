import React from 'react';
import { render } from 'enzyme';
import Empty from '../Empty';

describe('Empty', () => {
  test('renders correctly', () => {
    expect(render(<Empty theme={stubData.theme} />)).toMatchSnapshot();
  });
});
