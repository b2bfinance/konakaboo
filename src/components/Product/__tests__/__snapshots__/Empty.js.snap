// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty renders correctly 1`] = `
.c2 {
  width: 100%;
  min-height: 1px;
  -webkit-flex: 0 0 100%;
  -ms-flex: 0 0 100%;
  flex: 0 0 100%;
  max-width: 100%;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 100%;
}

.c1 {
  margin: 2rem auto;
  padding: 2rem;
}

@media (min-width:62em) {
  .c2 {
    -webkit-flex: 0 0 35%;
    -ms-flex: 0 0 35%;
    flex: 0 0 35%;
    max-width: 35%;
  }
}

<div
  class="c0"
>
  <div
    class="c1 c2"
  >
    <p>
      There are no product matching your requirements.
    </p>
  </div>
</div>
`;
